package.json

```diff
@@ -15,10 +15,10 @@
     "test:report": "npm run lint && npm run unit:report && npm run typescript",
     "test": "npm run lint && npm run unit && npm run typescript",
     "coverage": "npm run unit -- --cov --coverage-report=html",
-    "coveralls": "npm run unit -- --cov",
+    "test:ci": "npm run lint && npm run unit -- --coverage-report=html --no-browser && npm run typescript",
```

Comment:
https://github.com/ldapjs/node-ldapjs/blob/dd2a722169cfa0e780ccc53ae61784cda7ac00e1/package.json#L38

Reply:
thanks! I searched for it for more than 10min